docs/release-notes/release-notes-0.1.0.md

```diff
@@ -17,15 +22,11 @@ Environment variables can be set at compile time by specifying the variable and
 `PORT=4000 ./bin/elixir_node console`
 
 The following environment variables can be configured:
-* PORT - the port on which the Phoenix server will listen for any HTTP requests
-* SYNC_PORT - the port that will be used for any sync related P2P message receiving
-* ACCOUNTS_PATH - path to the JSON file containing any preset account balances
-* SIGN_KEYS_PASS - the password that will be used for private key encryption/decryption
-* PEER_KEYS_PASS - the password that will be used for peer private key encryption/decryption
-* PEER_KEYS_PATH - path to the peer key files
-* SIGN_KEYS_PATH - path to the sign key files
-* NEW_CANDIDATE_NONCE_COUNT - the number of mining attempts to be done before building a new block candidate
-* PERSISTENCE_PATH - path to Rox database
-
-It's possible to configure multiple environment variables, they should be separated with a space, for example:
-`PORT=4000 SYNC_PORT=3016 ./bin/elixir_node console`
+* PORT - the port on which the Phoenix server will listen for any HTTP requests, default is `4000`
```

Comment:
The default directories are incorrect. You need to specify full path. Which is automatically in the priv dir if it is not specified. Did you try this configuration?